drivers/pinmux/stm32/pinmux_stm32f0.h

```diff
@@ -54,6 +61,7 @@
 #define STM32F0_PINMUX_FUNC_PA4_SPI1_NSS \
 	(STM32_PINMUX_ALT_FUNC_0 | STM32_PUSHPULL_NOPULL)
 #define STM32F0_PINMUX_FUNC_PA5_SPI1_SCK \
+	(STM32_OSPEEDR_VERY_HIGH_SPEED) | \
```

Comment:
Nitpick: use of parenthesis looks odd. Can you fix it ?

Reply:
Yes. I wanted to do it in the simplest way possible.